iina/MainMenuActions.swift

```diff
@@ -367,9 +367,15 @@ extension MainMenuActionHandler {
     }
     let subURL = URL(fileURLWithPath: path)
     let subFileName = subURL.lastPathComponent
-    let destURL = currURL.deletingLastPathComponent().appendingPathComponent(subFileName, isDirectory: false)
     do {
+      // When streaming save the subtitles file to the user's ~/Movies directory, otherwise
+      // save the file to the same directory the video is in.
+      let destDirURL = player.info.isNetworkResource ? try FileManager.default.url(
```

Comment:
I feel like people who are streaming would rather not having subtitles randomly show up in ~/Movies. Can we use a temporary directory instead?

Reply:
For many of my submissions I've intentionally been in "only fix it" mode where I resist the temptation cleanup and refactor code and instead try and make the smallest change to just fix the issue. Always feel free to ask me to do more. This case is slightly different. I remember thinking the correct solution is to enhance the "Save Downloaded Subtitle" menu action to bring up a Save panel. The save in "Movies" was to avoid doing that. So rather than come up with some sort of temporary directory I'm thinking the correct solution is to prompt the user. Do you want me to try and make that change?

Reply:
I'm working on changing this to use a Save panel. Let me know if you think that is the wrong way to handle this.

Reply:
Ah, I just noticed that this was the codepath for saving subtitles, rather than just grabbing them and showing them when streaming. In that case, yes, please present a save dialog for these files.

Reply:
I got that limping yesterday. I will get that pushed today.

Reply:
The "Save Downloaded Subtitle" menu item has been changed to bring up a Save panel. I was surprised to not find good examples on the net for doing that and handling saving if there is an existing file. Definitely look over how I'm handling that. I updated the menu text to include an ellipsis. I copied the existing menu text into the corresponding Localization file so the title of the Save panel is localized. Bummer the text is duplicated. If there is a better way to do this, don't hesitate to make me change this.

Reply:
I just noticed that PRs that modify anything other than the english translations are rejected to avoid conflicts with the Crowdin process. Please let me know if I need to remove the changes to the main menu and localization files I added with the latest commit.